pkg/cmd/server/api/v1/types_test.go

```diff
@@ -230,6 +230,8 @@ oauthConfig:
     sessionMaxAgeSeconds: 0
     sessionName: ""
     sessionSecretsFile: ""
+  templates:
+    login: ""
```

Comment:
make sure a openshift-docs and openshift-ansible issue is open to track the new config value